packages/stream-model-handler/src/__tests__/model-handler.test.ts

```diff
@@ -788,137 +458,10 @@ describe('ModelHandler', () => {
     )
   })
 
-  it('throws error if changes metadata', async () => {
-    const genesisCommit = (await Model._makeGenesis(
-      context.api,
-      PLACEHOLDER_CONTENT
-    )) as SignedCommitContainer
-    await context.ipfs.dag.put(genesisCommit, FAKE_CID_1)
-
-    const payload = dagCBOR.decode(genesisCommit.linkedBlock)
-    await context.ipfs.dag.put(payload, genesisCommit.jws.link)
-
-    // apply genesis
-    const genesisCommitData = {
-      cid: FAKE_CID_1,
-      type: CommitType.GENESIS,
-      commit: payload,
-      envelope: genesisCommit.jws,
-    }
-    const state = await handler.applyCommit(genesisCommitData, context)
-
-    const state$ = TestUtils.runningState(state)
-    const doc = new Model(state$, context)
-    const rawCommit = doc._makeRawCommit(FINAL_CONTENT)
-    const newDid = 'did:3:k2t6wyfsu4pg0t2n4j8ms3s33xsgqjhtto04mvq8w5a2v5xo48idyz38l7zzzz'
-    rawCommit.header = { controllers: [newDid] }
-    const signedCommit = await Model._signDagJWS(context.api, rawCommit)
-
-    await context.ipfs.dag.put(signedCommit, FAKE_CID_2)
-
-    const sPayload = dagCBOR.decode(signedCommit.linkedBlock)
-    await context.ipfs.dag.put(sPayload, signedCommit.jws.link)
-
-    // apply signed
-    const signedCommitData = {
-      cid: FAKE_CID_2,
-      type: CommitType.SIGNED,
-      commit: sPayload,
-      envelope: signedCommit.jws,
-    }
-    await expect(handler.applyCommit(signedCommitData, context, state)).rejects.toThrow(
-      /Updating metadata for Model Streams is not allowed/
-    )
-  })
-
-  it('fails to apply commit with invalid prev link', async () => {
```

Comment:
some of these tests could in principle be updated to work with anchor commits and maintain the core of what the test was trying to check.  Up to you whether it's worth the effort or not

Reply:
Isn't this assured by other parts of the codebase anyway?